TestApp/android/app/src/main/java/com/testapp/MainApplication.java

```diff
@@ -42,7 +43,7 @@ public boolean getUseDeveloperSupport() {
                     new AppCenterReactNativePushPackage(MainApplication.this),
                     new AppCenterReactNativeCrashesPackage(MainApplication.this, getResources().getString(R.string.appCenterCrashes_whenToSendCrashes)),
                     new AppCenterReactNativeAnalyticsPackage(MainApplication.this, getResources().getString(R.string.appCenterAnalytics_whenToEnableAnalytics)),
-                    new AppCenterReactNativeDataPackage(),
+                    new AppCenterReactNativeDataPackage(MainApplication.this, true),
```

Comment:
Remove parameter here, using resources on other modules is old and legacy code and here it's hardcoded anyway.

If we need an option (which I don't believe we need now) we can use the appcenter-config.js file.

Reply:
Removed the boolean parameter